.npmrc

```diff
@@ -1,3 +1,4 @@
+enable-pre-post-scripts=true
```

Comment:
I'm not sure but is this line needed? `postinstall` works without this line.
https://github.com/pnpm/pnpm/issues/2891#issuecomment-698318201

Maybe you faced this one?
https://github.com/pnpm/pnpm/issues/4888#issuecomment-1155659767

Reply:
OK, I'll remove this line then.

```suggestion
```